v1.0/update.md

```diff
@@ -8,16 +8,399 @@ The `UPDATE` [statement](sql-statements.html) updates rows in a table.
+
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly    |
+|  3 | NULL     | stanley  |
+|  4 |   9000.0 | kelly    |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update All Rows
+
+{{site.data.alerts.callout_danger}}If you don't use the WHERE clause to specify the rows to be updated, the values for all rows will be updated.{{site.data.alerts.end}}
+
+~~~ sql
+> UPDATE accounts SET balance = 5000.0;
+
+> SELECT * FROM accounts;
+~~~ 
+~~~
++----+---------+----------+
+| id | balance | customer |
++----+---------+----------+
+|  1 |  5000.0 | michael  |
+|  2 |  5000.0 | kelly    |
+|  3 |  5000.0 | stanley  |
+|  4 |  5000.0 | kelly    |
++----+---------+----------+
+(4 rows)
+~~~
+
+### Update and Return Values
+
+In this example, the `RETURNING` clause returns the `id` values of the rows updated, which are generated server-side by the `unique_rowid()` function. The language-specific versions assume that you have installed the relevant [client drivers](install-client-drivers.html).
+
+{{site.data.alerts.callout_success}}This use of <code>RETURNING</code> mirrors the behavior of MySQL's <code>last_insert_id()</code> function.{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}When a driver provides a <code>query()</code> method for statements that return results and an <code>exec()</code> method for statements that don't (e.g., Go), it's likely necessary to use the <code>query()</code> method for <code>UPDATE</code> statements with <code>RETURNING</code>.{{site.data.alerts.end}}
+
+<section class="filters clearfix">
+    <button class="filter-button" data-scope="shell">Shell</button>
+    <button class="filter-button" data-scope="python">Python</button>
+    <button class="filter-button" data-scope="ruby">Ruby</button>
+    <button class="filter-button" data-scope="go">Go</button>
+    <button class="filter-button" data-scope="js">Node.js</button>
+</section>
+
+<section class="filter-content" markdown="1" data-scope="shell">
+<p></p>
+
+~~~ sql
+> UPDATE accounts SET balance = DEFAULT WHERE id = 1 RETURNING id;
+~~~
+
+~~~
++----+
+| id |
++----+
+|  1 |
++----+
+(1 row)
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="python">
+<p></p>
+
+~~~ python
+# Import the driver.
+import psycopg2
+
+# Connect to the "bank" database.
+conn = psycopg2.connect(
+    database='bank',
+    user='root',
+    host='localhost',
+    port=26257
+)
+
+# Make each statement commit immediately.
+conn.set_session(autocommit=True)
+
+# Open a cursor to perform database operations.
+cur = conn.cursor()
+
+# Insert two rows into the "accounts" table
+# and return the "id" values generated server-side.
+cur.execute(
+    'UPDATE accounts SET balance = DEFAULT WHERE id = 1 RETURNING id'
+)
+
+# Print out the returned values.
```

Comment:
Still looks the same. Seems like you haven't pushed your changes yet, for this and all other changes below?